meson since version ? converts backslashes to slashes, use other way to get pure filenames
--- AppStream-0.11.7/po/meson.build.orig	2017-10-22 19:36:07.000000000 +0200
+++ AppStream-0.11.7/po/meson.build	2017-12-23 14:05:05.250748125 +0100
@@ -21,7 +21,7 @@
         'find ' + meson.current_source_dir() +
         ' -type f' +
         ' -iname "*.po"' +
-        ' -printf \'%f\\n\'' +
+        ' -print | sed -e \'s,.*/,,\'' +
         ' | grep -oP \'.*(?=[.])\' | sort' +
         ' > ' + join_paths(meson.current_source_dir(), 'LINGUAS')
     ]
